src/mono/netcore/sample/HelloWorld/Makefile

```diff
@@ -2,8 +2,8 @@ TOP=../../../../../
 DOTNET:=$(TOP)./dotnet.sh
 DOTNET_Q_ARGS=--nologo -v:q -consoleloggerparameters:NoSummary
 
-MONO_CONFIG ?=Release
-MONO_ARCH=x64
+MONO_CONFIG ?=Debug
+MONO_ARCH=arm64
```

Comment:
No need to include your local testing changes to the netcore sample

Reply:
Yes, will revert the change to HelloWorld sample eventually.